### ISSUE TITLE
HW3: Question about comments in article detail page

### ISSUE THREAD

**Issue by clarkindy** [swsnu/swppfall2021#73]
In HW3 spec:

- Article detail page (`/articles/:id`)

  - You must have the following fields:

      | JSX Tag type | id |
      |--------------|----|
      | ... | ... |
      | button | edit-comment-button |
      | button | delete-comment-button |
      | ... | ... |
 
Does this mean that we have to include `edit-comment-button` and `delete-comment-button` for every comment available for the article? If it does, does it imply that our homework application does not follow HTML standard, which specifies that [every element must have a unique `id` attribute](https://html.spec.whatwg.org/multipage/dom.html#global-attributes:the-id-attribute-2)?

**Comment by kdh0102** [comment]
Yes, please include the buttons for all the comments of the author even though they have the same id.

**Comment by clarkindy** [comment]
Thank you for clarification!
